server/src/migrations/20191019160236_add_location_items.ts

```diff
@@ -1,8 +1,9 @@
+/* eslint-disable prefer-destructuring */
 import { Knex } from "knex";
 
 export async function up(knex: Knex): Promise<any> {
   const existingLocations = await knex.from("locations");
-  let defaultLocation;
+  let defaultLocation: any;
```

Comment:
Not that it super matters but if you look below you can what this will be.  It's a Location from the database

Reply:
Yeah ik that's what it is, it's just the typescript linter was giving me an error if I didn't include the `:any` so I just added it in there.